python/llm/test/basics/test_install.py

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
change the dir name to python/llm/test/packaging

Reply:
resolved.